introduction.md

```diff
@@ -2,3 +2,10 @@ Hi, my name is Jonatan.
 I'm a developer and love to fishing and climbing.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?